dev-requirements.txt

```diff
@@ -24,3 +24,4 @@ streamz
 sphinx==1.6.4
 sphinx_rtd_theme
 tifffile
+cloudpickle
```

Comment:
Can you move this into alphabetical order? Makes it easier to quickly check if a given entry is present.